teachertool/src/components/HeaderBar.tsx

```diff
@@ -151,27 +151,42 @@ export const HeaderBar: React.FC<HeaderBarProps> = () => {
             <></>
         );
         return (
-            <div className={css["user-menu"]}>
-                {teacherTool.userProfile ? (
-                    <MenuDropdown
-                        id="profile-dropdown"
-                        items={items}
-                        label={avatarElem || initialsElem}
-                        title={lf("Profile Settings")}
-                    />
-                ) : (
+            <>
+                <div>
```

Comment:
Is this div here in case we want to add stuff in the future?

Reply:
It wraps the button element so that reflow uses div-centric layout semantics. I just find it simpler to use divs for all layout. Mixing elements introduces a mishmash of different behavior, leading to more complicated css.

Reply:
Makes sense, thanks!